### PR TITLE
update warning message

### PR DESCRIPTION
**Pull request by SimonLab**
see https://github.com/…dwyl/dwylbot/pull/72#discussion_r122152421
update the message for the rule "no assignee with in-progress label"

### CODE REVIEW

**Review by nelsonic — APPROVED** [review]
@SimonLab 👍

File: web/controllers/rules/issue/noassignees.ex [hunk 1]
```diff
@@ -23,8 +23,8 @@ defmodule Dwylbot.Rules.Noassignees do
 
   defp error_message(login) do
     """
-    @#{login} the issue doesn't have anymore assignees but is still `in-progress`.
-    Please assign a user to this issue or remove the `in-progress` label.
+    :warning: @#{login} issue has no **assignee** but is still `in-progress`. Who is working on it?
+    Please assign a user to this issue or remove the `in-progress` label. Thanks! :heeart:
```
Comment (samhstn) [inline web/controllers/rules/issue/noassignees.ex]: `:heeart:` -> `:heart:`

**Review by samhstn — CHANGES_REQUESTED** [review]
Emoji typo :heeart: -> ❤️

**Review by samhstn — APPROVED** [review]
Fab ❤️

**Pull request merged**